app/models/user_preference.rb

```diff
@@ -17,5 +17,6 @@ class UserPreference < ApplicationRecord
   belongs_to :user
 
   validates :user, :associated => true
-  validates :k, :v, :length => 1..255, :characters => true
+  validates :k, :length => 1..255, :characters => true
+  validates :v, :length => 1..10_000, :characters => true
```

Comment:
Since this is intended to store arbitrary, computer-readable data: does it make sense to have this restriction to `characters`? Would it make sense to allow any binary bytes?